src/dvc_render/vega_templates.py

```diff
@@ -595,6 +568,38 @@ class SmoothLinearTemplate(Template):
                     "color": {"field": "rev", "type": "nominal"},
                 },
             },
+            {
+                "transform": [
+                    {
+                        "pivot": Template.anchor("group_by"),
+                        "value": "acc",
```

Comment:
This should be probably an anchor right?

Reply:
100%